darwin/utils.py

```diff
@@ -63,6 +67,12 @@ def is_deprecated_project_dir(project_path: Path) -> bool:
     return (project_path / "annotations").exists() and (project_path / "images").exists()
 
 
+def get_progress_bar(array: List, description: Optional[str] = None):
+    pbar = tqdm(array)
```

Comment:
Does `tqdm()` work with generators as well (not just lists)? If so, no reason to convert it to `List` before using it as an argument for `get_progress_bar`. Otherwise, it's fine.
Anyway, not blocking!